code/src/App.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react'
+import { Header } from './components/Header'
```

Comment:
You don't need brackets when importing only one thing
```suggestion
import Header from './components/Header'
```